extension/content.js

```diff
@@ -227,6 +228,32 @@ document.addEventListener('DOMContentLoaded', () => {
 		window.addEventListener('resize', infinitelyMore);
 	}
 
+	if (isProfile) {
```

Comment:
I think `isProfile` would match both user profiles and organisation profiles. You might want to check if this is indeed a user profile page based on the DOM.